src/lib/helpers/utils.ts

```diff
@@ -53,6 +55,10 @@ export function stringifyUnknown(unknown: unknown): string {
       return String(unknown);
     }
 
+    if (unknown instanceof AssertionError) {
+      return String(unknown);
+    }
```

Comment:
Replace thing above with
```suggestion
    if (unknown instanceof CLIError || unknown instanceof AssertionError) {
      return String(unknown);
    }
```